dist/pages/documentation.html

```diff
@@ -0,0 +1,157 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.7.1/styles/default.min.css">
+    <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/page.css">
+</head>
+
+<body>
+    <header class="page-header ">
+        <nav class="pB1 flex j-space-between a-items-center pT1 wrapper">
+            <div class="left-nav">
+                <ul class="flex j-space-between a-items-center">
+                    <li class="nav-links mR2 toggle-button" id="toggle"><a href="#"><i class="fa fa-bars"></i>
+                        </a></li>
+                    <li class="nav-links mR2"><a href="../../index.html">Home</a></li>
+                    <li class="nav-links mR2"><a href="./documentation.html">Docs</a></li>
+                </ul>
+            </div>
+            <div class="right-nav">
+                <ul class="flex j-space-between">
+                    <li class="nav-links link  mR1"><a href=""><i class="fas fa-search"></i></a></li>
+                    <li class="nav-links link mR1"><a href="https://github.com/manishrajmishra/"><i class="fab fa-github"></i></a></li>
+                    <li class="nav-links link"><a href=""><i class="far fa-moon"></i></a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+
+    <div class="wrapper page-layout">
+            <sidebar class="sidebar" id="sidebar">
+                <div class="sidebar-section">
+                    <h1 class="sidebar-title">Getting Started</h1>
+                    <ul class="sidebar-collection">
+                        <li class="sidebar-list"><a href="documentation.html">Introduction</a></li>
+                    </ul>
+                </div>
+
+                <div class="sidebar-section">
+                    <h1 class="sidebar-title">Layout</h1>
+                    <ul class="sidebar-collection">
+                        <li class="sidebar-list"><a href="containers.html">Containers</a></li>
+                        <li class="sidebar-list"><a href="grid-system.html">Grid System</a></li>
+
+                    </ul>
+                </div>
+
+                <div class="sidebar-section">
+                    <h1 class="sidebar-title">Basic Components</h1>
+                    <ul class="sidebar-collection">
+                        <li class="sidebar-list"><a href="components/badge.html">Badge</a></li>
+                        <li class="sidebar-list"><a href="components/avatar.html">Avatar</a></li>
+
+                        <li class="sidebar-list"><a href="components/buttons.html">Button</a></li>
+                        <li class="sidebar-list"><a href="components/image.html">Image</a></li>
+                        <li class="sidebar-list"><a href="components/alerts.html">Alerts</a></li>
+                        <li class="sidebar-list"><a href="components/tables.html">Tables</a></li>
+                        <li class="sidebar-list"><a href="components/card.html">Card</a></li>
```

Comment:
The whole list item gets highlighted on hover, but only on clicking on the text we can go to the page. That's an UX issue. 

And the colors are a bit on the non-visible side in this section.

Reply:
Okay i am changing what you have asked